docs/framework/network-programming/how-to-configure-network-tracing.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: "How to: Configure Network Tracing"
 ms.date: "03/30/2017"
-helpviewer_keywords: 
+helpviewer_keywords:
```

Comment:
The keywords contains app.config while the article itself doesn't mention it. Should the keyword be changed to machine.config ?

Reply:
It actually does mention the app.config file, so I think it's fine:

"The application or computer configuration file..."